Index: json/reader.inl
===================================================================
--- json/reader.inl	(revision 4034)
+++ json/reader.inl	(working copy)
@@ -320,7 +320,7 @@
             case 'n':      string.push_back('\n');    break;
             case 'r':      string.push_back('\r');    break;
             case 't':      string.push_back('\t');    break;
-            case 'u':      string.push_back('\u');    break; // TODO: what do we do with this?
+//            case 'u':      string.push_back('\u');    break; // TODO: what do we do with this?
             default: {
                std::string sMessage = std::string("Unrecognized escape sequence found in string: \\") + c;
                throw ScanException(sMessage, inputStream.GetLocation());
Index: json/writer.inl
===================================================================
--- json/writer.inl	(revision 4034)
+++ json/writer.inl	(working copy)
@@ -148,7 +148,7 @@
          case '\n':        m_ostr << "\\n";    break;
          case '\r':        m_ostr << "\\r";    break;
          case '\t':        m_ostr << "\\t";    break;
-         case '\u':        m_ostr << "\\u";    break; // uh...
+//         case '\u':        m_ostr << "\\u";    break; // uh...
          default:          m_ostr << *it;      break;
       }
    }
